meson.build

```diff
@@ -25,12 +25,19 @@ else
   dxvk_library_path = meson.source_root() + '/lib32'
 endif
 
-if meson.get_cross_property('winelib', false)
+code = '''#ifndef __WINE__
+#error 1
+#endif'''
```

Comment:
I'm sure this works, but looks a bit weird, might be better to keep the `winelib` cross-property around.

Reply:
That would defeat the purpose of these changes ;)
But yeah, looks a bit weird. I still think it's better to detect features, maybe we can improve the code chunk instead?

Reply:
The reason I think it's awkward is because we explicitly have to enable winelib builds using the cross files anyway, so we might as well tell the build system what we want instead of working around it using some awkward piece of code.

This isn't quite the same as the other changes which mostly avoid setting compiler and linker options in multiple different places.

Reply:
Just to clarify: The motivation is to get rid of all the custom stuff to be able to reuse general cross files.
Like my `.local/share/meson/cross/wine64`, which I can just use with any project using `meson --cross-file=wine64`.

Why, and not just use your bundled version? Because meson doesn't accept `CFLAGS=foo meson` for cross compilation. I have to edit cross files to change any flags...

But okay, let me drop that patch then. I can add `winelib=true' to my file instead, but it'll be project specific and not generic - and everyone has add it.

Reply:
I wasn't aware of this use case (tbh I *do* find it rather odd and probably no one else is ever going to do it, but whatever), it's probably okay to keep it then.

Reply:
Hehe, ok, I'll add the commit back, thanks!